xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-wysiwyg/xwiki-platform-realtime-wysiwyg-test/xwiki-platform-realtime-wysiwyg-test-docker/src/test/it/org/xwiki/realtime/wysiwyg/test/ui/RealtimeWYSIWYGEditorIT.java

```diff
@@ -707,7 +707,7 @@ void editSameMacro(TestReference testReference, TestUtils setup)
         // Focus the rich text area.
         secondTextArea.click();
         // Move the caret inside the information box.
-        secondTextArea.sendKeys(Keys.ARROW_UP, "two ");
+        secondTextArea.sendKeys(Keys.ARROW_UP, Keys.TAB, "two ");
```

Comment:
This doesn't seem good. Why do you need to press tab now? Where is the focus before pressing tab?

Reply:
The focus was on the title so an extra tab was needed. With the change described in https://github.com/xwiki/xwiki-platform/pull/3023#discussion_r1561239990, there's no need for this change anymore :)